builder/vmware/common/driver_fusion5.go

```diff
@@ -35,7 +35,7 @@ func NewFusion5Driver(dconfig *DriverConfig, config *SSHConfig) Driver {
 }
 
 func (d *Fusion5Driver) Clone(dst, src string, linked bool, snapshot string) error {
-	return errors.New("Cloning is not supported with Fusion 5. Please use Fusion 6+.")
+	return errors.New("linked clones are not supported on this version")
```

Comment:
I would remain explicit that Fusion 5 is the issue here
```suggestion
	return errors.New("linked clones are not supported on VMWare Fusion 5, please upgrade")
```

Reply:
Resolved in 67054cd.